src/modules/enrollment/components/EnrollmentAssessmentActionButtons.tsx

```diff
@@ -55,35 +70,25 @@ const NewAssessmentMenu: React.FC<Props> = ({ enrollment }) => {
     const topItems: NavMenuItem[] = [];
     const bottomItems: NavMenuItem[] = [];
 
-    // Edge case: show "intake" item if the client is entered but does not have an intake
-    if (!enrollment.intakeAssessment) {
-      topItems.push({
-        key: 'intake',
-        to: getPath(AssessmentRole.Intake),
-        title: 'HUD Intake Assessment',
-      });
-    }
-
-    // Exit/Update/Annual can only be added to open enrollment
-    if (!enrollment.exitDate) {
-      topItems.push({
-        key: 'exit',
-        to: getPath(AssessmentRole.Exit),
-        title: 'HUD Exit Assessment',
+    if (!data?.client) return [];
+
+    const hudRoleTitles: Partial<Record<AssessmentRole, string>> = {
+      INTAKE: 'HUD Intake Assessment',
+      UPDATE: 'New HUD Update Assessment',
+      EXIT: 'New HUD Exit Assessment',
+      ANNUAL: 'New HUD Annual Assessment',
+      POST_EXIT: 'New HUD Post-Exit Assessment',
```

Comment:
```suggestion
      INTAKE: 'Intake Assessment',
      UPDATE: 'Update Assessment',
      EXIT: 'Exit Assessment',
      ANNUAL: 'Annual Assessment',
      POST_EXIT: 'Post-Exit Assessment',
```

Reply:
These seem to the the titles we get from the forms on the backend. Looks like we don't need to override the titles at all in this case.